acceptance/framework/consul/helm_cluster.go

```diff
@@ -201,9 +201,9 @@ func (h *HelmCluster) Destroy(t *testing.T) {
 			}
 		}
 	}
-	// Retry a few times because sometimes certain resources (like PVC) take time to delete
+	// Retry because sometimes certain resources (like PVC) take time to delete
 	// in cloud providers.
-	retry.RunWith(&retry.Counter{Wait: 1 * time.Second, Count: 10}, t, func(r *retry.R) {
+	retry.RunWith(&retry.Counter{Wait: 1 * time.Second, Count: 60}, t, func(r *retry.R) {
```

Comment:
I was wondering after merging this if 10 seconds was enough, and now I recall that its more like 10 minutes sometimes for these things to sort themselves :-/ 
Should we retry 600 times or just retry every 10 seconds, 60 times?
```suggestion
	retry.RunWith(&retry.Counter{Wait: 1 * time.Second, Count: 600}, t, func(r *retry.R) {
```